client/reader.js

```diff
@@ -74,18 +74,26 @@ rl.on('line', (line) => {
     }
   } else if (line.indexOf('FLEX: ') > -1) {
     address = line.match(/FLEX:.*?\[(\d*?)\] /)[1].trim();
-    if (line.match( /( ALN | GPN | NUM)/ )) {
-      message = line.match(/FLEX:.*?\[\d*\] ... (.*?)$/)[1].trim();
-      trimMessage = message;
-    } else {
-      message = false;
-      trimMessage = '';
+      if (line.match( /1600\/2\/F\/A/ )) {
+       if (line.match( /( ALN | GPN | NUM)/ )) {
+        frag = line.match(/FLEX:.*?\[\d*\] ... (.*?)$/)[1].trim();
```

Comment:
So frag needs to be declared further up - should be declared above line 54 (outside the `rl.on`). If it's declared within the `rl.on` block then it'd be overwritten on every line. Just adding `var frag;` up around line 40 should be all that's needed.